backend/dynamo-db/template.yaml

```diff
@@ -39,8 +41,8 @@ Resources:
           Projection:
             ProjectionType: ALL
 
-    # checkov:skip=CKV_AWS_28: "Ensure Dynamodb point in time recovery (backup) is enabled"
-    # checkov:skip=CKV_AWS_119: "Ensure DynamoDB Tables are encrypted using a KMS Customer Managed CMK"
+    # check:skip=CKV_AWS_28: "Ensure Dynamodb point in time recovery (backup) is enabled"
```

Comment:
I think these will need to be 'inside' the resource; so above `Type` and at the same level of indentation as `Type`.

See how https://github.com/alphagov/di-onboarding-self-service-experience/blob/main/backend/dynamo-api/template.yaml#L19 has nested them.  Failing that you'll have to RTM to figure it out.

Reply:
That's spot on @JohnWattsGDS. It is passing now.